styleguide/global/index.jade

```diff
@@ -0,0 +1,50 @@
+.container.standard-padding
+  .row.half-padding-full
+    .col-md-12
+      h4.red.no-margin Welcome to the
+      h2.no-margin Global Theme
+      h4.red.no-margin Let's get started
+
+  .row.half-padding-full
+    .col-md-12
+      p If you only want the global elements without any of ecommerce's patterns, css and js then this is the link for you. It is brand approved, includes bootstrap and is ready to go.
+      code.language-javascript
+              = '<link href="https://043a96a8a2caadd9c57463-9701ce6e6cbd7003c5a8a25a5a69546c.ssl.cf5.rackcdn.com/v1.1/global/css/global.css" />'
+              <br/>
```

Comment:
Visually it looks a little strange.  
![image](https://cloud.githubusercontent.com/assets/5263371/16826310/ff4f1920-4941-11e6-83ae-11096a185398.png)
![image](https://cloud.githubusercontent.com/assets/5263371/16826321/0e10402e-4942-11e6-8e43-360de3b5be2c.png)

There seems to be padding at the beginning of the first line and at the end of the second line only.  

I wonder if it would make more sense to have two separate 

`code.language-javascript` blocks and have those separated by a `br` instead? 

(also use jade syntax for br, not html)